packages/react-components/src/components/ErrorBar.tsx

```diff
@@ -248,3 +296,38 @@ const errorsToShow = (
     return error.timestamp > dismissal.dismissedAt;
   });
 };
+
+const messageBarType = (errorType: ErrorType): MessageBarType => {
+  switch (errorType) {
+    case 'callNetworkQualityLow':
+    case 'callNoSpeakerFound':
+    case 'callNoMicrophoneFound':
+    case 'callMicrophoneAccessDenied':
+    case 'callMicrophoneMutedBySystem':
+    case 'callMacOsMicrophoneAccessDenied':
```

Comment:
Haha, different messages for different system, good one

Reply:
Yeah, unfortunate but true.